File: python/model/category_blocking.py
from dataclasses import dataclass
from typing import List


@dataclass(frozen=True)
class CategoryBlocking:
    """配信制御情報が格納されているクラス"""

    advertiser_id: int  # 広告主ID
    category_ids: List[int]  # カテゴリIDの配列。配信タイプの許可/拒否情報は対応する広告主に格納

File: golang/model/category_blocking.go
package model

// 配信制御情報が格納されているクラス。
type CategoryBlocking struct {
	AdvertiserId uint   // 広告主ID
	CategoryIds  []uint // カテゴリIDの配列。配信タイプの許可/拒否情報は対応する広告主に格納
}
